pypesto/objective/amici.py

```diff
@@ -372,6 +379,20 @@ def call_unprocessed(
         """
         x_dct = self.par_arr_to_dct(x)
 
+        # only ask amici to compute required quantities
```

Comment:
Go to full reporting if return of rdatas is requested? Would also work for time trajectories without the changes below.

Reply:
Oh, good point.

Reply:
Wait. You mean `return_dict==True`? That is only handled in `ObjectBase`. `AmiciObjective` doesn't ever get so see that. Would have to pass that into all descendants of `ObjectBase` then. Do we want that?

Reply:
Okay, handled it via __call__

Reply:
neat!